.nvmrc

```diff
@@ -0,0 +1 @@
+9.3.0
```

Comment:
Dont think we can go this high on the version. The dockerfile specs node 8 and 8.9.3 is LTS. Is there something we need from 9.*?

Reply:
I just upgraded to latest after it complained. I will try LTS.